recipes/client.rb

```diff
@@ -18,6 +18,14 @@
 # limitations under the License.
 #
 
+nt_version = ::Windows::VersionHelper.nt_version(node)
+
+if nt_version == 6.1
```

Comment:
Couldn't we just use node['platform_version'] here and avoid calling the helper?